src/applications/common/schemaform/fields/ArrayField.jsx

```diff
@@ -289,7 +304,7 @@ export default class ArrayField extends React.Component {
                 'usa-button-disabled': !this.props.formData
               }
             )}
-            disabled={!this.props.formData}
+            disabled={!this.props.formData || isEmpty(this.props.formData[lastIndex])}
```

Comment:
This disables the "add another" button when no information has been provided, and prevents the creation of empty items (particularly when no fields are required by the schema/no validation prevents this from occuring).